sympy/core/parameters.py

```diff
@@ -65,7 +65,11 @@ def __setattr__(self, name, value):
             clear_cache()
         return super(_global_parameters, self).__setattr__(name, value)
 
-global_parameters = _global_parameters(evaluate=True, distribute=True)
+global_parameters = _global_parameters(\
+    evaluate=True,\
+    distribute=True,\
+    evaluate_matrix=False\
```

Comment:
These global flags are problematic and already don't work properly. We should not add new global flags.

In any case the argument for having evaluate does not apply to evaluate_matrix because evaluate is used for *disabling* automatic evaluation. If you want to make the matrix expressions evaluate you just have to call `.doit()`.

Reply:
I agree with Oscar, but as another note, the backslashes here are not needed and should be omitted.

Reply:
You're right. I will remove global_parameters.evaluate_matrix.